src/main/java/io/prometheus/cloudwatch/CloudWatchCollector.java

```diff
@@ -42,13 +22,28 @@
 
 import org.yaml.snakeyaml.Yaml;
 
+import software.amazon.awssdk.auth.credentials.AwsCredentialsProvider;
+import software.amazon.awssdk.regions.Region;
+import software.amazon.awssdk.services.cloudwatch.CloudWatchClient;
+import software.amazon.awssdk.services.cloudwatch.CloudWatchClientBuilder;
+import software.amazon.awssdk.services.cloudwatch.model.*;
```

Comment:
This one isn't explicit.

Reply:
Done : https://github.com/prometheus/cloudwatch_exporter/pull/307/commits/c7fd02fba98cc0bca1cc30e13510be148c647388